### PR TITLE
Bug 2114681: Add verification on kernel param rcutree.kthread

### PR DESCRIPTION
**Pull request by shaior**
Automates https://bugzilla.redhat.com/show_bug.cgi?id=2114681

### CODE REVIEW

File: test/e2e/performanceprofile/functests/1_performance/rt-kernel.go [hunk 1]
```diff
@@ -71,4 +76,25 @@ var _ = Describe("[performance]RT Kernel", func() {
 		err = nodes.HasPreemptRTKernel(&nonPerformancesWorkers[0])
 		Expect(err).To(HaveOccurred(), "Node should have non-RT kernel")
 	})
+
+	It("[test_id:rctree] Should have kernel param rcutree.kthread", func() {
```
Comment (yanirq) [inline test/e2e/performanceprofile/functests/1_performance/rt-kernel.go:80]: test id should be replaced eventually

Comment (marioferh) [inline test/e2e/performanceprofile/functests/1_performance/rt-kernel.go:80]: Here we have another test to check kernel boot args:
https://github.com/openshift/cluster-node-tuning-operator/blob/a7fcf60362a2dd11519055d865df1782263854ff/test/e2e/pao/functests/1_performance/performance.go#L122

It is necessary to do it in a new file? Could it be added there?

Comment (shaior) [reply test/e2e/performanceprofile/functests/1_performance/rt-kernel.go:80]: yes np, will do in the PR targeting master branch.

**Comment by shaior** [comment]
new PR targeting master branch:
https://github.com/openshift/cluster-node-tuning-operator/pull/464

**Pull request closed without merging**